glfw/src/time.zig

```diff
@@ -58,9 +58,11 @@ pub inline fn setTime(time: f64) void {
 
     std.debug.assert(!std.math.isNan(time));
     std.debug.assert(time >= 0);
-    // TODO: Look into why GLFW uses this hardcoded float literal as the maximum valid value for 'time'.
-    // Maybe propose upstream to name this constant. 
-    std.debug.assert(time <= 18446744073.0);
+    // assert time is lteq to largest number of seconds representable by u64 with nanosecond precision
+    std.debug.assert(time <= max_time: {
+        const @"2^64" = std.math.maxInt(u64);
```

Comment:
Ah I forgot Zig doesn't have an integer power operator, we could also use `2 << 63` here.

Reply:
That's true, but I thought I'd try and use the type "u64" directly in the code to indicate it is involved, thus `maxInt`.